Assets/Scripts/EnemyController.cs

```diff
@@ -136,11 +144,24 @@ void Blobify()
         animator.SetBool("IsDead", true);
         isBlobified = true;
         agent.stoppingDistance = 0;
-        agent.destination = gameObject.FindClosestObject(keyLocations).transform.position;
+        if (isAreaEnemy)
+            agent.destination = assignedKeyLocation.transform.position;
+        else
+        {
+            agent.destination = new Vector3(Random.Range(0f, 1f), 0, Random.Range(0f, 1f)); // adjust this
```

Comment:
Can we just use the closest key location here?

Reply:
idea is that if the enemy is "progress enemy", it has to go really far to that key location even if it is the closest one. So with Mikkel we thought to just make them go somewhere randomly for a bit and make them disappear